NeoMathEngine/include/NeoMathEngine/NeoMathEngine.h

```diff
@@ -1184,6 +1184,11 @@ class NEOMATHENGINE_API IMathEngine : public IDnnEngine {
 	virtual void SetReuseMemoryMode( bool enable ) = 0;
 	virtual CMemoryHandle HeapAlloc( size_t count ) = 0;
 	virtual void HeapFree( const CMemoryHandle& handle ) = 0;
+	// Transfers memory handle from other thread owner to this thread.
+	// Caution! Do not use this method directly, only through the method CDnnBlob::TransferDataToThisThread()
+	// CDnnBlob can be transfered, if at least one pointer to it is still exist, and on other side
```

Comment:
The first two lines are good. The next two have the same problem as above (but here the original description is cool so you can simply remove the thing about pointers and about threads being finished)

Reply:
fixed